crates/handlers/src/graphql/mod.rs

```diff
@@ -44,7 +46,22 @@ use rand::{thread_rng, SeedableRng};
 use rand_chacha::ChaChaRng;
 use sqlx::PgPool;
 use tracing::{info_span, Instrument};
+use ulid::Ulid;
+
+mod model;
+mod mutations;
+mod query;
+mod state;
 
+#[cfg(test)]
+pub use state::BoxState;
+pub use state::State;
```

Comment:
I really like using `self::` when doing relative imports, so that it doesn't clash with a potential crate called `state`. wdyt?